ziti-fabric/subcmd/set_fingerprint.go

```diff
@@ -0,0 +1,92 @@
+package subcmd
+
+import (
+	"fmt"
+	"github.com/michaelquigley/pfxlog"
+	"github.com/openziti/foundation/storage/boltz"
+	"github.com/spf13/cobra"
+	"go.etcd.io/bbolt"
+	"regexp"
+	"strings"
+)
+
+func init() {
+	Root.AddCommand(setRouterFingerprintCmd)
+}
+
+var setRouterFingerprintCmd = &cobra.Command{
+	Use:   "fingerprint <db-file> <router-id> <fingerprint>",
+	Short: "Manually specify the router client fingerprint, the controller must not be running.",
```

Comment:
if the controller is running it'll just block forever?

Reply:
The DB lock can't be acquired and it will fail.